pkg/arvo/lib/ph/azimuth.hoon

```diff
@@ -266,24 +266,29 @@
 ++  dawn
   |=  [who=ship seed=(unit seed:able:jael)]
   ^-  dawn-event
-  =/  spon
+  =/  spon=(list [ship point:azimuth])
+    |-  ^-  (list [ship point:azimuth])
     =/  =ship  (^sein:title who)
-    =/  spon-spon
-      ?:  ?=(%czar (clan:title ship))
-        [| ~zod]
-      [& (^sein:title ship)]
-    =/  life-rift  ~|([ship lives] (~(got by lives) ship))
-    =/  =life  lyfe.life-rift
-    =/  =rift  rut.life-rift
-    =/  =pass
-      %^    pass-from-eth:azimuth
-          (as-octs:mimes:html (get-public ship life %crypt))
-        (as-octs:mimes:html (get-public ship life %auth))
-      1
-    :^    ship=ship
-        *[address address address address]:azimuth
-      `[life=life pass rift spon-spon ~]
-    ~
+    =/  a-point=[^ship point:azimuth]
+      =/  spon-spon
+        ?:  ?=(%czar (clan:title ship))
+          [| ~zod]
```

Comment:
Is this just a trick to get `:aqua` to work correctly?

Reply:
I thought galaxies had default values for sponsor on chain, but it looks like they actually have a sponsor of themselves.  Shouldn't make a difference; I'll change it.